### ISSUE TITLE
HR3 C1152 netcdfb tarball too big to be saved on HPSS 

### ISSUE THREAD

**Issue by jiandewang** [NOAA-EMC/global-workflow#2339]
### What is wrong?

in HR3 C1152 runs *atmos*nc file is too large (38G each), plus *sfcf*nc (4G each), the netcdfb tarball will be 5.7T for 16 days run, and this surpasses the HPSS tarball max size. Below is the archiving step log information:
_HTAR: a   gfs.20200210/00/model_data/atmos/history/gfs.t00z.atmf231.nc
HTAR: a   gfs.20200210/00/model_data/atmos/history/gfs.t00z.sfcf231.nc
ERROR: hpss_WriteList error -27, file offset 3298534883328 - aborting
HTAR: HTAR FAILED
###WARNING  htar returned non-zero exit status._

Unfortunately rocoto and script won't give non-zero return code in the arching step so the workflow will run from end to end without error.

A sample log file can be found on dogwood: /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20200210/COMROOT/2020021000/HR3-20200210/logs/2020021000/gfsarch.log (see line 7907)

### What should have happened?

netcdfb tarball should contains all atmf and sfcf files

### What machines are impacted?

WCOSS2

### Steps to reproduce

clone HR3 tag
setup C1152 16 days run
check archive step log file

### Additional information

n/a

### Do you have a proposed solution?

split to several sub-tarball

**Comment by jiandewang** [comment]
I think the max tarball size on HPSS is a bit more than 3T (based on my observations from the HR3b run HPSS tar file)

**Comment by JessicaMeixner-NOAA** [comment]
I think we have two issues here: 
* 1. needing to split the tar-ball into multiple 
* 2. Understanding why the job doesn't fail even those htar failed.  Ideally wouldn't we want that message to be captured and the job to fail in this case?

**Comment by JessicaMeixner-NOAA** [comment]
We have an issue for the second part here: https://github.com/NOAA-EMC/global-workflow/issues/2242

**Comment by WalterKolczynski-NOAA** [comment]
Yes, this issue will just be for splitting the tarball up. Handling htar failures will be handled separately.

**Comment by KateFriedman-NOAA** [comment]
> I think the max tarball size on HPSS is a bit more than 3T (based on my observations from the HR3b run HPSS tar file)

The recommendation I was told years ago is to limit to 2T but we should try to avoid making tarballs that large. Perhaps stay closer to 1T.

**Comment by JessicaMeixner-NOAA** [comment]
An update on this... 

We split the tarring of atmf*.nc files into 4 tar-balls and the sfcf*.nc into a 5th tar-ball.  However, this consistently is timing out.  

@aerorahul you mentioned a longer term fix for the archive job.  Will this have restart capability or be able to be multiple jobs? I think our short-term solution will be break-up the atmf*.nc archiving into a second job temporarily to run HR3b but long-term we'll definitely need to re-think (including if these files need to be archived).

**Comment by aerorahul** [comment]
@JessicaMeixner-NOAA 
I cannot yet tell whether there will be individual jobs or a single sequential job without running tests and gathering more information like what are the file sizes, number of archive tar balls, etc.
A restart capability is technically not required for archive jobs as they are not part of the operational suite.  Ops run their own archive jobs.

**Comment by JessicaMeixner-NOAA** [comment]
Right now, we cannot archive everything currently requested in a single job for C1152 and a 16 day run on WCOSS2.   I've currently split out the atmf*.nc files archiving temporarily into a second file. This is being tested now and I'll report back.  

Good to know about the restart feature! Thanks for sharing.

**Comment by aerorahul** [comment]
@jiandewang  
Have we recently encountered this issue with HR5 or tests at C1152 resolution?
Or should we consider this issue resolved?

**Comment by JessicaMeixner-NOAA** [comment]
I have not noticed issues with the C1152 retrotests. I think this can be closed.

**Comment by aerorahul** [comment]
> I have not noticed issues with the C1152 retrotests. I think this can be closed.

Thanks for the quick reply @JessicaMeixner-NOAA and confirming things are running smoothly.  If issues come up, we can reopen this or another issue.

**Comment by jiandewang** [comment]
this is resolved as since HR3 we have compression on atm * nc files which greatly reduced the files size.
